src/components/experiment-tracking/runs-list/runs-list.scss

```diff
@@ -4,7 +4,7 @@
   bottom: 0;
   left: 0;
   overflow-y: scroll;
-  position: absolute;
+  // position: absolute;
```

Comment:
nit - maybe we can remove the comments?

Reply:
Of course. This is leftover. Great catch. Removed!